docs/miscellaneous.rst

```diff
@@ -394,12 +394,14 @@ The following is the order of precedence for operators, listed in order of evalu
 +============+=====================================+============================================+
 | *1*        | Postfix increment and decrement     | ``++``, ``--``                             |
 +            +-------------------------------------+--------------------------------------------+
-|            | Function-like call                  | ``<func>(<args...>)``                      |
+|            | New expression                      | ``new <typename>``                         |
 +            +-------------------------------------+--------------------------------------------+
 |            | Array subscripting                  | ``<array>[<index>]``                       |
 +            +-------------------------------------+--------------------------------------------+
 |            | Member access                       | ``<object>.<member>``                      |
 +            +-------------------------------------+--------------------------------------------+
+|            | Function-like call                  | ``<func>(<args...>)``                      |
```

Comment:
I looked into the code because I was pretty sure that `new` bound tightly to the typename. Turns out `A.value` is interpreted as a qualified typename, not member access, which is why you need to add the parenthesis.